packages/gestalt-datepicker/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "gestalt-datepicker",
+  "version": "2.2.1",
+  "license": "Apache-2.0",
+  "homepage": "https://pinterest.github.io/gestalt",
+  "description": "A set of React UI components which enforce Pinterest’s design language",
```

Comment:
@AlbertCarreras let's improve this description and make it date picker specific.

Reply:
fixed